modules/op-limiting-secret-access-to-specific-steps.adoc

```diff
@@ -0,0 +1,10 @@
+// This module is included in the following assembly:
+//
+// *openshift-docs/cicd/pipelines/authenticating-pipelines-using-git-secret.adoc
+
+[id="op-limiting-secret-access-to-specific-steps_{context}"]
+= Limiting secret access to specific steps
+
+By default, the supported secrets for {pipelines-shortname} are stored in the `$HOME/tekton/home` directory, and are available for all steps in a task.
+
+To limit a secret to be accessible only by specific steps, explicitly specify a volume using the secret definition, and manually mount the volume into the specific steps.
```

Comment:
```suggestion
To limit a secret to specific steps, use the secret definition to specify a volume, and mount the volume in specific steps.
```

Reply:
As a user reading this topic summary, I might like to have detailed steps that to show me how to perform the procedure.

Reply:
@nikhil-thomas  @concaf - This is a very valid point raised by Rolfe. I adapted the content available [upstream](https://github.com/tektoncd/pipeline/blob/main/docs/auth.md#limiting-secret-access-to-specific-steps), and perhaps we can provide steps to mount volumes, or point the reader to a page with similar information. Help me with this, please!